roles/rsyslog/roles/input_roles/ovirt/vars/main.yaml

```diff
@@ -1,5 +1,11 @@
+# oVirt rpm packages
+
+# __rsyslog_ovirt_prereq_packages
+#
+# List of prerequired rpm packages for oVirt
+__rsyslog_ovirt_prereq_ packages: []
```

Comment:
is there a space in the name?

Reply:
Ouch.  Thanks for finding it out...  Fixed.